cmd/snapctl/flags.go

```diff
@@ -43,6 +43,16 @@ var (
 		Name:  "running",
 		Usage: "Shows running plugins",
 	}
+	flPassword = cli.BoolFlag{
+		Name:  "password, p",
+		Usage: "Password for REST API authentication",
+	}
+	flPasswordPath = cli.StringFlag{
+		Name:   "Password, P",
```

Comment:
Better yet, I see below that the env is called `SNAPCTL_PASS_PATH`, let's call this `"password file path, P"`.